src/model/presentation/xml/SVGFile.java

```diff
@@ -14,10 +14,13 @@
 import java.util.List;
 
 public class SVGFile implements Presentation{
-
+	
+	/************************************
+	 *	Attributes
+	 ************************************/
 	private String fileName;
 	private String xmlcontent;
-	private ArrayList<Shape> shapes = new ArrayList<Shape>();
+	private ArrayList<Object> shapes = new ArrayList<Object>();
```

Comment:
Because it can be a Shape or a a SVGFile (Presentation).
I thought of creating a new interface that would regroup both class, but it was a little to much I thought